### DISCUSSION TITLE
Qual o melhor teste técnico que você já fez?

### DISCUSSION THREAD

**Discussion by luisfmsouza** [frontendbr/forum#2073] [Discussão]
Fala galera, estava conversando com um amigo sobre os testes/entrevistas técnicas que rolam nos processos seletivos. Geralmente envolvem essa etapa do processo codar um projeto do zero ou usam alguma ferramenta automatizada (tipo HarckerHank), não conseguimos concordar em qual é o melhor do ponto de vista dos candidatos.

A pergunta que ficou dessa conversa é: na sua opinião, qual o melhor teste técnico que você já fez?

**Comment by rafaelangical** [comment]
Na minha opinião todo teste é válido, agora o melhor teste é aquele simples porém funcional, onde com pouco código escrito o dev consegue expressar bem sua lógica e sua forma de codar.

**Reply by luisfmsouza** [reply]
Tem um exemplo de algum teste que vc tenha feito que se encaixa nessa descrição, @rafaelangical ?
